app/page.tsx

```diff
@@ -26,38 +29,86 @@ export default function Home() {
 
   const UnderConstructionModal = () => {
     return (
-      <Modal title="Under Construction" open={isModalOpen} onOk={handleOk} onCancel={handleCancel} footer={null}>
+      <Modal
+        title="Under Construction"
+        open={isModalOpen}
+        onOk={handleOk}
+        onCancel={handleCancel}
+        footer={null}
+      >
         <div className="flex items-center flex-col gap-5">
-          <Image
-            src={UnderConstruction}
-            width={200}
-            alt='Under Maintenance'
-          />
-          <div className="text-2xl font-light text-center">Purrintables is under construction. Some links might be broken. In the mean time you can message us over at <a href="https://www.instagram.com/purrintables" target="_blank">Instagram</a> or <a href="https://www.facebook.com/purrintables" target="_blank">Facebook</a>!</div>
+          <Image src={UnderConstruction} width={200} alt="Under Maintenance" />
+          <div className="text-2xl font-light text-center">
+            Purrintables is under construction. Some links might be broken. In
+            the mean time you can message us over at{" "}
+            <a href="https://www.instagram.com/purrintables" target="_blank">
+              Instagram
+            </a>{" "}
+            or{" "}
```

Comment:
Can we avoid {" "}? Let's just use normal spaces. If unavoidable let's use "flex gap-3" or something similar.

Reply:
sorry the {" "} where added by prettier, im gonna figure out how do avoid these wehn using auto formatting